webapp-django/crashstats/settings/__init__.py

```diff
@@ -1,8 +1,6 @@
 from .base import *
 try:
     from .local import *
-except ImportError, exc:
-    exc.args = tuple(
-        ['%s (did you rename settings/local.py-dist?)' % exc.args[0]]
-    )
-    raise exc
+except ImportError:
+    print ('WARNING - no local.py'
+           ' (deprecated, use environment variables instead)')
```

Comment:
I mean; if it succeeds to import `local.py` you're potentially stuck in the old way of doing things. And you need to re-organize yourself away from `local.py` to a new `.env` file instead.